api-reference/v2/api-reference-guide.md

```diff
@@ -0,0 +1,213 @@
+- For "update" operations: `Updates a {resource}.`
+- For "delete" operations: `Removes a {resource}.` 
+
+Ensure that the description here is consistent with that listed in the resource reference.
+
+### "Endpoint" section
+
+Use this section to define the API endpoint.
+
+**Tips**:
+
+- Use fenced code blocks (a pair of three backticks `` ``` ``) to make your code distinctive from other text blocks.
+- Replace {METHOD} with the actual request method and capitalize all letters. For example, `POST`.
+- In the {request_url} segment, start with a slash character `/` and only provide the URL path (the segment after the hostname). The base URL can be omitted if you already document it in the API overview. For example, `/v2/users`.
+- If the {request_url} contains path variables, use a placeholder to indicate the variable name. The format of placeholders should be consistent throughout the documentation set and conform to your organization’s guidelines. As an option, you can use snake case characters in curly brace `{}`, delimited by underscores. For example, `{user_id}`.
+- Optionally use a different color to make the path parameters easily identifiable.
+- Do not add slash characters `/` at the end.
+
+### "Description" section
+
+Use this section to provide more information on what the endpoint does, the purpose, and use cases of this API endpoint.
+
+Optionally add notes about the API versioning information, API limit, deprecation status, and whether a replacement is available. If they are not applicable, remove the paragraphs.
+
+### "Authorization" section
+
+Provide a link to the common `Authorization` section in the API reference overview.
+
+If calling the endpoint requires additional user roles or permissions, document them in this section.
+
+### "Request schema" section
+
+Use this section to define the specifications of the endpoint.
+
+Each of the sub-sections is optional and should be adopted according to the actual endpoint definition:
+
+- **Path parameters**: parameters defined within the path of the endpoint, denoted by placeholders.
+- **Query parameters**: appended to the end of the request URL, after a question mark `?`. Parameters and their assigned values are connected by an equal sign `=`. Multiple query parameters are delimited by an and sign `&`.
+- **Header parameters**: often the same across endpoints in an API set. Include this section only when specific header parameters are required for this endpoint.
+- **Request body**: only applicable for requests using methods that permit a payload, such as POST, PUT, and PATCH. Include a link to the description of the resource type if applicable.
+
+If no parameters or request body can be sent in the request, specify "None" in this section.
+
+**Tips**:
+
+- In each of the tables, keep the parameter name the same as what is presented in the endpoint section above.
+- In the `Required/Optional` column, specify "REQUIRED" or "Optional" to avoid ambiguity.
```

Comment:
```suggestion
- In the `Required/Optional` column, specify "Required" or "Optional" to avoid ambiguity.
```
Question: Is there an established convention that we are following to use capitalization for REQUIRED? If not, I'd err toward using lower case "Required".